src/Microsoft.Extensions.StackTrace.Sources/PortablePdbReader.cs

```diff
@@ -20,6 +20,11 @@ internal class PortablePdbReader : IDisposable
 
         public void PopulateStackFrame(StackFrameInfo frameInfo, MethodBase method, int IlOffset)
         {
+            if (method.Module.Assembly.IsDynamic)
```

Comment:
cc @davidfowl - @pakrym found an issue where accessing `Assembly.Location` throws for dynamically generated assemblies.